web/src/app/services/task/task.service.ts

```diff
@@ -39,6 +39,8 @@ export type TaskUpdate = {
 export class TaskService {
   private tasks$: Observable<List<Task>>;
 
+  private addLoiTaskId = '_addLoi';
```

Comment:
Nits:
* Let's define at top level as a `const`
* Let's use `$addLoi` instead of `_addLOi`, which is more ergonomic imo.